python/kfserving/requirements.txt

```diff
@@ -16,3 +16,5 @@ cloudevents>=1.2.0
 avro>=1.10.1
 boto3>=1.17.32
 botocore>=1.20.32
+ray[serve]>=1.3.0
```

Comment:
1.4.0 was just released so feel free to bump the version :-)

Reply:
@simon-mo Looks like 1.4.0 has some breaking change regarding namespaces, so I pinned to 1.3.0 for now

Reply:
Hi @yuzisun, you can specify a namespace inside ray.init(..., namespace="..."). See 
https://ray--16786.org.readthedocs.build/en/16786/serve/deployment.html#id3

cc @edoakes